.env.example

```diff
@@ -0,0 +1,21 @@
+PORT=8081
```

Comment:
Did you set the environment variables in your project folder? You should set these envs and remove this example file

Reply:
Yes, I set the environment variables for the project. I already delete the example file.